tests/checks/integration/test_nginx.py

```diff
@@ -22,27 +22,42 @@ def setUp(self):
                 {
                     'nginx_status_url': 'http://localhost:44441/nginx_status/',
                     'tags': ['first_one'],
+                    'nginx_ssl_verify': True,
                 },
                 {
                     'nginx_status_url': 'http://dummyurl:44441/nginx_status/',
                     'tags': ['dummy'],
+                    'nginx_ssl_verify': True,
                 },
                 {
                     'nginx_status_url': 'http://localhost:44441/nginx_status/',
                     'tags': ['second'],
+                    'nginx_ssl_verify': True,
```

Comment:
None of these tests actually are made against an https:// url, nor do they flip the flag from True to False.
Is that expected?